src/main/java/com/cs56fitnessapp/models/Meal.java

```diff
@@ -1,29 +1,154 @@
 package com.cs56fitnessapp.models;
+import java.util.ArrayList;
+import java.time.LocalDate;
+
 
 /**
- * @author Nerissa Hsieh
+ * @author Nerissa Hsieh & Jimwell Castillo
  * Date Created: 10/03/17
- * Date Modified: 10/03/17
+ * Date Modified: 10/20/17
  */
 
-public class Meal implements DietFacts {
+public class Meal implements DietFacts{
+    private LocalDate date;
+    private MealTitle title;
+    private ArrayList <FoodEntry> foodEntriesList;
+
+    //    Old Constructor,commented out
+    //    public Meal(LocalDate date, String title, ArrayList <FoodEntry> foodEntriesList, MealTitle title1) {
+    //        this.date = date;
+    //        this.title = title1;
+    //        this.foodEntriesList = new ArrayList<>();
+    //     }
+
+    /**
+     * Edited 10.20.17, Jim
+     * **/
+
     @Override
     public int getCaloriesIn() {
+        for(int i = 0; i<this.foodEntriesList.size(); i++){
+            FoodEntry food = this.foodEntriesList.get(i);
+            if(true){
+                return getCaloriesIn();
+            }
+        }
         return 0;
     }
```

Comment:
Hint you should use this line of code 
`this.foodEntriesList.get(i).getCalories();` 

Will need to fix logic in a following way:
1. declare a variable of type int `totalCalories` on top of the for loop, initialize it to 0
2. update this variable every iteration
3. return this variable

Same goes for other overridden methods.